tests/test-criticisms/test_ppc_plots.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import matplotlib
+matplotlib.use('Agg', warn=False)
```

Comment:
I wasn't sure how necessary this line is, but I didn't know much about testing plotting functions, and so I took a look at what PyMC3 had done and took this from there.